### PR TITLE
config: pipeline: Fix failures caused by unsuccessful checkouts

### PR DESCRIPTION
**Pull request by musamaanjum**
- Some branches have disappeared. Remove those branches.
- Some branches have been renamed.

There was for-kernelci branch present in peterz's tree which has disappeared. I'm replacing some other branch for now until we get a reply from him.

Close: https://github.com/kernelci/kernelci-pipeline/issues/888

### CODE REVIEW

File: config/pipeline.yaml [hunk 1]
```diff
@@ -3071,10 +3071,6 @@ build_configs:
     tree: amlogic
     branch: 'for-next'
 
-  amlogic_integ:
-    tree: amlogic
-    branch: 'integ'
```
Comment (musamaanjum) [inline config/pipeline.yaml:3076]: Sent email asking the Neil about it. Let's wait before merging the PR.

File: config/pipeline.yaml [hunk 2]
```diff
@@ -3345,7 +3333,7 @@ build_configs:
 
   peterz:
     tree: peterz
-    branch: 'kernelci'
+    branch: 'perf/core'
```
Comment (musamaanjum) [inline config/pipeline.yaml:3336]: Sent email asking the Peter about it. Let's wait before merging the PR.

Comment (musamaanjum) [reply config/pipeline.yaml:3336]: Sent reminder

Comment (nuclearcat) [reply config/pipeline.yaml:3336]: I think remove tree and branches for now, if we get feedback we will add tree back

Comment (musamaanjum) [reply config/pipeline.yaml]: Sure.

**Pull request merged**